src/stories/Library/hero/Hero.tsx

```diff
@@ -5,13 +5,15 @@ import { ReactComponent as ArrowLargeRight } from "../Arrows/icon-arrow-ui/icon-
 
 export type HeroProps = {
   image: string;
+  type?: string;
```

Comment:
Could this be something like 
contentType or similar in order to be more descriptive. 

Type is very generic imo.